DTML.EduBot/Common/MessageTranslator.cs

```diff
@@ -0,0 +1,101 @@
+﻿namespace DTML.EduBot.Common
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Net.Http;
+    using System.Net.Http.Headers;
+    using System.Configuration;
+    using System.Threading.Tasks;
+    using System.Web;
+    using System.Xml.Linq;
+
+    public class MessageTranslator
+    {
+        /// <summary>
+        /// Get authentication token for translator API
+        /// </summary>
+        /// <param name="key">OcpApimSubscriptionKey</param>
+        /// <returns>token if the request is successful null otherwise</returns>
+        public static async Task<string> GetAuthenticationTokenAsync(string key)
```

Comment:
This can be private. The get authentication token would always be called from methods in MessageTranslator.